front/app/modules/commercial/insights/admin/components/StyledTextComponents.tsx

```diff
@@ -12,3 +12,8 @@ export const SectionTitle = styled.h2`
 export const TooltipContent = styled.p`
   font-weight: normal;
 `;
+
+export const TooltipContentList = styled.ul`
```

Comment:
Do you really need this? Maybe you can use our `Text` component `as="ul"`? Also the one on the top is probably no longer needed and can be replaced by Text as well.